manifests/v1beta1/installs/katib-standalone/katib-config.yaml

```diff
@@ -45,7 +45,7 @@ runtime:
       image: docker.io/kubeflowkatib/suggestion-enas:latest
       resources:
         limits:
-          memory: 200Mi
+          memory: 400Mi
```

Comment:
After I upgraded the Python version, I faced OOM killed in the ENAS suggestion service.